libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -1,31 +1,67 @@
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
 import { RouterTestingModule } from '@angular/router/testing';
 import { MockDirective } from 'ng-mocks';
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+import { IonicModule } from '@ionic/angular';
 
-import { PageComponent } from './page.component';
+import { DesignTokenHelper } from '../../helpers/design-token-helper';
+import { TestHelper } from '../../testing/test-helper';
+import { PageComponent, PageContentComponent } from './page.component';
 import { FitHeadingDirective } from '../../directives/fit-heading/fit-heading.directive';
 
+const size = DesignTokenHelper.size;
+
 describe('PageComponent', () => {
-  let component: PageComponent;
-  let fixture: ComponentFixture<PageComponent>;
+  let spectator: SpectatorHost<PageComponent>;
+  let element: HTMLElement;
+  let ionToolbar: HTMLElement;
+
+  const dummyContent = `<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci animi aperiam deserunt dolore error esse
```

Comment:
Why the departure from https://baconipsum.com/ ?

![giphub](https://media1.giphy.com/media/l3vRhaxVcQ1i0CZlC/giphy.gif)